### PR TITLE
Enable Prow's slack client (just on test-infra for now)

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @spxtr @brahmaroutu

### CODE REVIEW

**Comment by kargakis** [comment]
You also need to add the `slackevents` plugin under test-infra.

File: prow/cmd/hook/main.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var (
 	githubTokenFile = flag.String("github-token-file", "/etc/github/oauth", "Path to the file containing the GitHub OAuth secret.")
 
 	webhookSecretFile = flag.String("hmac-secret-file", "/etc/webhook/hmac", "Path to the file containing the GitHub HMAC secret.")
-	slackTokenFile    = flag.String("slack-token-file", "", "Path to the file containing the Slack Kubernetes Team Token.")
+	slackTokenFile    = flag.String("slack-token-file", "/etc/slack/token", "Path to the file containing the Slack token to use.")
```
Comment (spxtr) [inline prow/cmd/hook/main.go]: Default to empty string here.

File: prow/plugins.yaml [hunk 2]
```diff
@@ -14,6 +14,15 @@ heart:
   adorees:
   - k8s-merge-robot
 
+slackevents:
+  - repos:
```
Comment (spxtr) [inline prow/plugins.yaml]: Remove one level of indent.

File: prow/plugins.yaml [hunk 3]
```diff
@@ -14,6 +14,15 @@ heart:
...
 
+slackevents:
+- repos:
+  - kubernetes/test-infra
+  channels:
+  - sig-testing
+  whitelist:
+  - cjwagner
+
+
```
Comment (spxtr) [inline prow/plugins.yaml]: extra newline

File: prow/plugins.yaml (hunk 2 above)
Comment (cjwagner) [reply prow/plugins.yaml]: done

File: prow/plugins.yaml (hunk 3 above)
Comment (cjwagner) [reply prow/plugins.yaml]: done

**Comment by spxtr** [comment]
LGTM as well. For this one you *should* bump hook.

File: prow/Makefile [hunk 4]
```diff
@@ -15,7 +15,7 @@
 all: build test
 
 
-HOOK_VERSION             ?= 0.157
+HOOK_VERSION             ?= 0.158
```
Comment (spxtr) [inline prow/Makefile]: Also hook_deployment.yaml.

**Pull request merged**